content/nginxaas-azure/getting-started/ssl-tls-certificates/ssl-tls-certificates-portal.md

```diff
@@ -16,6 +16,36 @@ You can manage SSL/TSL certificates for F5 NGINXaaS for Azure (NGINXaaS) using t
 
 {{< include "/nginxaas-azure/ssl-tls-prerequisites.md" >}}
 
+## Finding the Azure Key Vault Secret Identifier
+
+When adding a certificate using the Azure CLI, Terraform, or ARM/Bicep templates, you need to provide the **Secret Identifier** from Azure Key Vault.
+
+{{< call-out "important" >}}
+**Use the Secret Identifier, not the Certificate Identifier.**
+
+When you create a certificate in Azure Key Vault, it creates three related objects:
```

Comment:
I'm thinking we should not explicitly call this out given that we also support workflows where users can import their TLS certs/keys into AKV (rather than creating them in AKV) - I suspect this is the more common use case. My suggestion would be to leave out lines 26-31.